src/register/index.js

```diff
@@ -99,12 +101,39 @@ const hideAllUsernameErrors = () =>
     .querySelectorAll('span')
     .forEach(message => (message.style.display = 'none'));
 
-const changeUsernameError = errorId => {
+const changeUsernameError = id => {
   hideAllUsernameErrors();
-  document.getElementById(errorId).style.display = 'flex';
+  document.getElementById(id).style.display = 'flex';
+};
+
+const checkForError = async inputs => {
+  for (const input of inputs) {
+    if (!input.checkValidity()) return true; // Input value is invalid
+    if (input.id === 'username') {
```

Comment:
Maybe it would be better to have these ids as some constant somewhere, instead of this magic-string
Same for ln 120

Reply:
Very good point! I'll fix this later on!